core/src/apps/common/storage/slip39.py

```diff
@@ -0,0 +1,73 @@
+from micropython import const
+
+from apps.common import storage
+from apps.common.storage import slip39_mnemonics
+
+# fmt: off
+# App:
+_SLIP39                    = const(0x02)  # SLIP-39 namespace
+# Keys:
+_SLIP39_IN_PROGRESS        = const(0x00)  # bool
+_SLIP39_IDENTIFIER         = const(0x01)  # bytes
+_SLIP39_THRESHOLD          = const(0x02)  # int
+_SLIP39_REMAINING          = const(0x03)  # int
+_SLIP39_WORDS_COUNT        = const(0x04)  # int
+_SLIP39_ITERATION_EXPONENT = const(0x05)  # int
+# fmt: on
+
+
+def set_slip39_in_progress(val: bool):
```

Comment:
I think you don't need to include "_slip39_" in the function names here and in slip39_mnemonics.py, because you then get things like storage.slip39.get_slip39_identifier(), which is rather tedious.

Reply:
Great point! Fixed.